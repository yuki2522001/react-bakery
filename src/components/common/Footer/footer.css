.footer {
  margin-top: 100px;
  background-color: var(--border-btn);
  padding: 10px;
}

.footer__contact {
  text-align: center;
}