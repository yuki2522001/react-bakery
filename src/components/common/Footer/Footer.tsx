import React from "react";
import "./footer.css";

const Footer: React.FC = () => {
  return (
    <footer className="footer">
      <p data-testid="footer" className="footer__contact">
        Copyright © 2022 — Bakery WordPress Theme by Milingona.
      </p>
    </footer>
  );
};

export default Footer;
