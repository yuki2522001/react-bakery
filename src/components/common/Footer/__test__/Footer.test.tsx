import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Footer from "../Footer";

describe("Footer component", () => {
  test("should render p tag", () => {
    const { getByTestId } = render(<Footer />);
    expect(getByTestId("footer")).toBeInTheDocument();
  });

  test("matches snapshot", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
})