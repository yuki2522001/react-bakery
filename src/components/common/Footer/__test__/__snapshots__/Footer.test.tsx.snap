// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <p
      class="footer__contact"
      data-testid="footer"
    >
      Copyright © 2022 — Bakery WordPress Theme by Milingona.
    </p>
  </footer>
</DocumentFragment>
`;
